### ISSUE TITLE
SQL Syntax in Python Regex faults on single quotes in a comment

### ISSUE THREAD

**Issue by benjolitz** [sublimehq/Packages#167]
The following python code disables SQL syntax highlighting due to a bug in the regex in handling single quotes:

```
query = \
    '''SELECT
        (
        SELECT CASE field
            WHEN 1
            THEN -- comment's say that
                EXISTS(
                select 1)
            ELSE NULL
        ) as result
    '''
```

Here is a screen shot of the bug:
![screen shot 2016-02-12 at 1 37 51 pm](https://cloud.githubusercontent.com/assets/270476/13021100/0d537bd2-d18e-11e5-9a35-58cf0309f8be.png)

Here is a screen shot of what it should look like:

![screen shot 2016-02-12 at 1 37 39 pm 1](https://cloud.githubusercontent.com/assets/270476/13021102/127d8fc6-d18e-11e5-926f-42afcdb4c1c8.png)

**Comment by benjolitz** [comment]
Amendment:

Anything that ends with a single quote instead of a double quote provokes the syntax highlighting bug

**Comment by wbond** [comment]
This should be fixed by 143166ec8367d95f60311fb80a1db073872578f9
